app/javascript/controllers/application.js

```diff
@@ -13,8 +13,14 @@ application.register('popover', Popover);
 application.register('toggle', Toggle);
 application.register('slideover', Slideover);
 
+
 // Configure Stimulus development experience;
 application.debug = true;
 window.Stimulus   = application;
 
+// import Flatpickr
+import Flatpickr from 'stimulus-flatpickr';
```

Comment:
I'm wondering, since we're planning on using flatpickr soon anyway, when it makes sense to remove since the work is done.

Reply:
kind of confused about this comment?

Reply:
so I finished implementing the inclusion of flatpickr here, which was slated as a task on the repo (see #1 ). I expect I'll be wanting to reach for it when I build the next set of features, so am wondering whether I need to take it out of this PR here.